apple/internal/processor.bzl

```diff
@@ -475,7 +478,8 @@ def _bundle_post_process_and_sign(
         process_and_sign_template,
         provisioning_profile,
         rule_descriptor,
-        rule_label):
+        rule_label,
+        env = None):
```

Comment:
[question] Why default value to None? Can this be mandatory?

Reply:
Just to make code look consistent, because I added default value for `_processor`. I suppose, because it's private method, we can make it mandatory here.

Reply:
Yep. Could you make all places mandatory 🙏 None of the parameters are optional in the function, would prefer to stick to than consistency better :D